picky-asn1-x509/src/pkcs7/signed_data.rs

```diff
@@ -73,13 +126,69 @@ impl<'de> de::Deserialize<'de> for CertificateSet {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub enum CertificateChoices {
+    Certificate(Asn1RawDer),
+    Other(picky_asn1_der::Asn1RawDer),
```

Comment:
Asn1RawDer is already in scope:

```suggestion
    Other(Asn1RawDer),
```